docs/tutorials/tutorial-a-guide-to-using-the-snowplow-inspector-extension/index.md

```diff
@@ -0,0 +1,173 @@
+
+Selecting registries in the list will filter the schema listing to show only the schemas they contain.
+You can also search for schemas using the field at the top of the directory.
+Use the ‘Clear Filters’ button to clear these and see the full list of schemas again.
+
+Using the ‘Registries...’ button on the right, you can:
+
+- Add more custom registries
+- Edit existing registries you've previously configured
+- Remove configured registries so they aren't accessible to the extension any more
+- Import a list of registries from an existing [Iglu Resolver configuration](/docs/pipeline-components-and-applications/iglu/iglu-resolver/index.md)
+
+Selecting a "local" registry will activate the ‘Schemas...’ button and allow you to manage the schemas it contains.
+
+When adding or editing a registry, the following types are supported:
+1. [Local](#local-registries)
+1. [Data Structures API](#data-structures-api-registries)
+1. [Iglu Server](#iglu-server-registries)
+1. [Static HTTP](#static-registries)
+
+Configuring a registry requires a name and one of these types.
+You may also optionally include a priority, and list of vendor prefixes.
+Each type of registry may require additional configuration like API keys to function.
+
+Once configured, schemas from the registry should load in the ‘Manage Schemas’ view, and events using those schemas will be checked for validation.
+
+Now that the extension can query your private Iglu repositories the extension will be able to tell you in real time whether your events and entities are failing validation from entirely within the browser.
+An example of an event passing validation is shown below:
+
+![A Self Describing Event called 'example_event' is displayed in the Snowplow Inspector extension. It contains a property 'example_field_1' with the value 'abc'. The extension says the event is Valid against its schema, signalled by green text.](images/event-passing-validation.png)
+
+An example of an event failing validation is shown below, the reason for this event failing validation is due to “example_field_1” being specified as allowing only the "string" type, meaning it cannot be null.
+Hovering over the "Invalid" warning displays text describing which aspect of the payload failed the validation.
+Clicking the warning copies this error to your clipboard.
+
+![A Self Describing Event called 'example_event' is displayed in the Snowplow Inspector extension. It contains a property 'example_field_1' with a null value. The extension says the event is Invalid against its schema, signalled by a bright red background.](images/event-validation-failed.png)
+
+This is useful, but when it comes to the debugging of failed events and the investigation of exactly WHY a particular event has failed validation there are additional tools that should be considered.
```

Comment:
```suggestion
This is useful, but when it comes to the debugging of failed events and the investigation of exactly _why_ a particular event has failed validation there are additional tools that should be considered.
```

Reply:
What are those additional tools? We leave the reader hanging

Reply:
Woops; this was left over from the original page, intended to remove it.